config/config_test.go

```diff
@@ -95,3 +96,24 @@ func TestConfigNewLoggerConfig(t *testing.T) {
 		t.Errorf("Config.NewLoggerConfig() = %v, want %v", got, want)
 	}
 }
+
+func TestConfigNewKafkaConfig(t *testing.T) {
+	v, err := NewConfig("testConfigForViper", "./testData/")
+	if err != nil {
+		t.Fatalf("Cant start test, err: %v", err)
+	}
+
+	var partition int32
+	partition = 1
```

Comment:
Is it necessary to declare variable in such way? Or can't it be const in that case?)

Reply:
fixed